### DISCUSSION TITLE
removing the linked notes in the top-level folder navigation

### DISCUSSION THREAD

**Discussion by purashy** [TriliumNext/Trilium#4497] [Q&A] [answered]
I apologize if this question was asked before. But I tried to look in here and did not know what to look for exactly.

**So if I have:**

```
Note
     sub-note 1
     sub-note 2
```

Basically removing the linked notes under the Top-level notes/folder.

When clicking on `Note`, it shows `[sub-note 1] [sub-note 2]` at the buttom of whatever I have written in `Note`

Picture:

![image](https://github.com/zadam/trilium/assets/4239073/92b21581-d2ad-4cf6-9ed0-b8c7e4cfe240)

To elaborate, I want to hide: `Grocery list for today`, `Book to read` and `The Last Question`.

Is this possible? Thank you in advance.

**Comment by rodrigoalexandrecosta** [comment, marked as answer]
Yes, this is possible, you can add the label `#hideChildrenOverview` to your Inbox note.

**Reply by purashy** [reply]
You sir, are a freaking legend ❤️ thank you.

For those curious:

![image](https://github.com/zadam/trilium/assets/4239073/9c0b5429-79fc-4a75-9a43-d2a21527e366)
